### DISCUSSION TITLE
whichkey menu for C-w window navigation broken?

### DISCUSSION THREAD

**Discussion by routingincircles** [LunarVim/LunarVim#2647] [Q&A]
In previous releases of LunarVim there was a whichkey menu when hitting C-w, like this:

![grafik](https://user-images.githubusercontent.com/83008348/169648381-26fb19fb-4ae9-460b-af00-f610c0ef8bab.png)

I just installed LunarVim on a different workstation, using the stable "no alarm no surprise" way described on lunarvim.org, and got a new version (master-aa4c993), where that menu doesn't appear any more. Is that intended, or should I open an issue for that?

Also, on a side note, I was surprised not to get the latest release (1.1.3) but something newer when installing the "stable" version. So if it's stable, shouldn't it be a new release then?

**Comment by glatavento** [comment]
I added this in my config and now it works again
```lua
for _, preset in ipairs({ "z", "g", "nav", "motions", "windows", "operators", "text_objects" }) do
  lvim.builtin.which_key.setup.plugins.presets[preset] = true
end
```
